.github/workflows/sniff.yaml

```diff
@@ -0,0 +1,71 @@
+name: Sniff stage - quality control checks
+
+on:
+    pull_request:
+        branches: [master, develop]
+    push:
+        branches: [master]
+
+jobs:
+    qa_checks:
+        name: Quality control checks
+        runs-on: ubuntu-latest
+        env:
```

Comment:
I don't feel strongly about removing these, but because there is only one branch of PHPCS and WPCS used in this workflow, these environment variables could be removed entirely in favor of just hard-coding them below in the commands.

Reply:
Good point. For the sniff stage, there is basically only one check, so I can just hardcode them.

Reply:
Thinking this through the next day, the only argument for leaving them is consistency across workflows. If the same `env` variables are actually needed in other workflow files, I could see leaving them in all workflows for consistency's sake.